src/main/io/serial_cli.c

```diff
@@ -971,25 +975,25 @@ static void cliSerial(char *cmdline)
                 if (baudRateIndex < BAUD_9600 || baudRateIndex > BAUD_115200) {
                     continue;
                 }
-                portConfig.msp_baudrateIndex = baudRateIndex;
+                portConfig.baudRates[BAUDRATE_MSP_SERVER] = baudRateIndex;
```

Comment:
with baudrates array, this may be rewritten as `const struct { min, max, auto} baudLimits[4] = ...` and for loop